README.md

```diff
@@ -1,70 +1,60 @@
-# Getting Started with Create React App
+# Weather Forecast
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+ This web application fetches weather forecast data from https://open-meteo.com/en/docs API for cities displayed on the homepage, the user can also choose specific country and then the city to view cooresponding temperature data.
 
-## Available Scripts
+ ## Live link
+ [Link](https://elastic-turing-eb476d.netlify.app/)
 
-In the project directory, you can run:
+ ## Screenshot
+<img src='./src/laptop-screenshot.png'>
+<img src='./src/mobile-screenshot.png'>
 
-### `yarn start`
+## Presentation video
+[video](https://youtu.be/nw5vNCaCKn8)
```

Comment:
- [ ] In the video you are required to speak about a piece of code that you find interesting.

Reply:
Thanks for the feedback, I think the presentation video isn't meant to get technical, it has to be very clear to people with no coding background.